doc/grpc_security_audit_response.md

```diff
@@ -0,0 +1,62 @@
+# Addressing grpc_security_audit
+
+Please read grpc_security_audit.pdf for background on identified gRPC security issues.
+
```

Comment:
It'll be good add a paragraph about GRP-01-001 stating that it has already been fixed and point to the relevant PRs. That way resolution for all the issues in the report will be in one place.

Reply:
I reorganized security_audit.md a bit to address this. I actually forgot about that doc. I've removed the new doc I added and put the text in the other doc instead